File: Java/Bryann Valderrama/Geometria/area_Circuncirculo.java
class Main {

    static double PI = 3.14159265;

    public static double area_cicumscribed(double a) {
        return (a * a * (PI / 3));
    }

    public static void main(String[] args) {
        double a = 6.0;
        System.out.println("Area de circulo circunscrito: " + area_cicumscribed(a));
    }
}

File: Codigos estudiantes por lenguaje/PY/Bryann Valderrama/Geometria/areaCircuncirculo.py
from sys import stdin, stdout
import math
rl = stdin.readline
wr = stdout.write


def area_circumscribed(a):
    return (a * a * (math.pi / 3))


a = int(rl())
wr(f'Area de circulo circunscrito: {area_circumscribed(a)}')
